### PR TITLE
Add all step content > #57

### LINKED ISSUE #57: Add all Content
**Related issues**
#56 needs to be completed first 

**Content doc to use**
https://docs.google.com/document/u/1/d/1Fq7vB5LZW8lachRyZZcPg9_ZDaYsmHxEaOv_somyf6s/edit?usp=drive_web&ouid=114879239832987647932

---

### Acceptance Criteria:

_REMEMBER THAT WHOEVER WORKS ON THIS ISSUE MUST TICK OFF ALL THE POINTS IN THIS LIST UNLESS THERE IS CLEAR AGREEMENT IN THE COMMENTS TO SAY OTHERWISE. **DO NOT REVIEW A PR INVOLVING THIS ISSUE UNLESS THIS HAS BEEN DONE**_ 

- [x] Set up content data folder 
- [x] Add all content as part of npm run build:db (or whatever command gets created in package.json) 

**Note: This isn't pulling it into the front end yet, just ensuring all required content is in the database**

### PR DESCRIPTION
**Pull request by thejoefriel**
#### Issues this PR relates to
Closes #57 

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

You will need to run either production build or dummy build 

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/database/dummy-data-prod/stepsContent.js [hunk 1]
```diff
@@ -0,0 +1,538 @@
...
+        ],
+      },
+    ],
+    whatYouWillNeedToKnow: [
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+    ],
```
Comment (ashatat) [inline server/database/dummy-data-prod/stepsContent.js:169]: Just check that this is meant to be empty like that as there is no mentioning it in the doc file

File: server/database/dummy-data/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import dotenv from 'dotenv';
 import createUsers from './users';
 import createOrganisations from './organisations';
+import createLandingPage from '../dummy-data-prod/landingPageContent';
```
Comment (ashatat) [inline server/database/dummy-data/index.js:4]: I think we've agreed to make all files/folders should be lowercase with dashes/hyphens 
I know it's not the best, but at least we should make things consistent across the app
```suggestion
import createLandingPage from '../dummy-data-prod/landing-page-content';
```
also I think you forgot to createSteps here, unless it was intentional

File: server/database/dummy-data/index.js [hunk 3]
```diff
@@ -9,7 +10,8 @@ if (process.env.NODE_ENV !== 'production') {
 const buildData = async () => {
   const createdData = {};
   createdData.users = await createUsers(createdData);
-  createdData.users = await createOrganisations(createdData);
+  createdData.organisations = await createOrganisations(createdData);
```
Comment (ashatat) [inline server/database/dummy-data/index.js:13]: good catch 👍

**Review by ashatat — CHANGES_REQUESTED** [review]

File: server/database/dummy-data-prod/stepsContent.js [hunk 4]
```diff
@@ -0,0 +1,538 @@
...
+    whatYouWillNeedToKnow,
+    topTip,
+    otherTips,
+  ]);
+  return res.rows[0];
+};
+
+const addStepContent = async () => {
+  const eligibilityContent = {
+    stage: 'BEFORE_CLAIMING',
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply server/database/dummy-data-prod/stepsContent.js]: good point - done

File: server/database/dummy-data-prod/stepsContent.js [hunk 5]
```diff
@@ -0,0 +1,538 @@
...
+    stepOrder: 1,
+    title: 'Wait! Should I apply?',
+    description: `Let's find out with a quick and easy benefit calculator`,
+    pageTitle: `Should you claim?`,
+    pageDescription: `Using the benefit calculator enables you to find out an estimate of how much Universal Credit you may be entitled to.`,
+    howLongDoesItTake: { timeRangeText: `15 to 25 minutes` },
+    WhereDoYouNeedToGo: {
+      link:
+        'https://www.entitledto.co.uk/benefits-calculator/Intro/Home?cid=0af743fb-414d-4559-a0c9-b88d26a88671',
+      type: 'LINK',
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply server/database/dummy-data-prod/stepsContent.js]: done

File: server/database/dummy-data-prod/stepsContent.js (hunk 1 above)
Comment (thejoefriel) [reply server/database/dummy-data-prod/stepsContent.js]: good spot - fixed

File: server/database/dummy-data/index.js (hunk 2 above)
Comment (thejoefriel) [reply server/database/dummy-data/index.js]: done

**Review by ashatat — APPROVED** [review]

**Pull request merged**